tracing/src/main/java/com/palantir/tracing/DetachedSpan.java

```diff
@@ -64,6 +64,22 @@ static DetachedSpan start(
         return Tracer.detachInternal(observability, traceId, parentSpanId, operation, type);
     }
 
+    /**
+     * Marks the beginning of a span, which you can {@link #complete} on any other thread.
+     *
+     * @see DetachedSpan#start(String)
+     */
+    @CheckReturnValue
+    static DetachedSpan start(
+            Observability observability,
+            String traceId,
+            Optional<String> requestId,
```

Comment:
This method should be unused, the system can generate a new requestId if this is a SERVER_INCOMING span.

Reply:
delete